src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/PreviewPage.cs

```diff
@@ -14,6 +14,16 @@ public PreviewPage(IWebDriver driver, ITestOutputHelper helper) : base(driver, h
         {
         }
 
+        /// <summary>
+        /// Awful, will get replaced with a proper method once preview page contains a way to go to dashboard.
+        /// </summary>
+        public void NavigateToDashboard()
```

Comment:
Could be replaced with 'driver.Navigate().Back()`

Reply:
Would then be good to wait for something found on the dashboard to be displayed after calling the `Back()` method

Reply:
Good idea, done and done.